x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -413,11 +406,8 @@ class IndexActionsContextMenuUi extends Component {
             >
               <p>
                 <FormattedMessage
-                  id="xpack.idxMgmt.indexActionsMenu.deleteEntity.deleteEntityWarningDescription"
-                  defaultMessage="
-                    This operation cannot be undone. Make sure you have appropriate
-                    backups.
-                  "
+                  id="xpack.idxMgmt.indexActionsMenu.deleteIndex.deleteEntityWarningDescription"
```

Comment:
Can we change "deleteEntityWarningDescription" to "deleteWarningDescription" since we're removing references to "entity" elsewhere?